psi4/driver/procrouting/proc_table.py

```diff
@@ -179,31 +187,39 @@
         'scf'           : proc.run_scf_hessian,
     },
     'properties' : {
-        'hf'       : proc.run_scf_property,
-        'scf'      : proc.run_scf_property,
-        'mp2'      : proc.select_mp2_property,
-        'cc2'      : proc.run_cc_property,
-        'ccsd'     : proc.run_cc_property,
-        'eom-cc2'  : proc.run_cc_property,
-        'eom-ccsd' : proc.run_cc_property,
-        'detci'    : proc.run_detci_property,  # full control over detci
-        'cisd'     : proc.run_detci_property,
-        'cisdt'    : proc.run_detci_property,
-        'cisdtq'   : proc.run_detci_property,
-        'ci'       : proc.run_detci_property,  # arbitrary order ci(n)
-        'fci'      : proc.run_detci_property,
-        'rasscf'   : proc.run_detci_property,
-        'casscf'   : proc.run_detci_property,
-        'omp2'     : proc.select_omp2_property,
-        'omp2.5'   : proc.select_omp2p5_property,
-        'omp3'     : proc.select_omp3_property,
-        'olccd'    : proc.select_olccd_property
-        # Upon adding a method to this list, add it to the docstring in driver.properties
+        'hf'           : proc.run_scf_property,
+        'scf'          : proc.run_scf_property,
+        'mp2'          : proc.select_mp2_property,
+        'cc2'          : proc.run_cc_property,
+        'ccsd'         : proc.run_cc_property,
+        'eom-cc2'      : proc.run_cc_property,
+        'eom-ccsd'     : proc.run_cc_property,
+        'detci'        : proc.run_detci_property,  # full control over detci
+        'cisd'         : proc.run_detci_property,
+        'cisdt'        : proc.run_detci_property,
+        'cisdtq'       : proc.run_detci_property,
+        'ci'           : proc.run_detci_property,  # arbitrary order ci(n)
+        'fci'          : proc.run_detci_property,
+        'rasscf'       : proc.run_detci_property,
+        'casscf'       : proc.run_detci_property,
+        'omp2'         : proc.select_omp2_property,
+        'omp2.5'       : proc.select_omp2p5_property,
+        'omp3'         : proc.select_omp3_property,
+        'olccd'        : proc.select_olccd_property
```

Comment:
```suggestion
        'olccd'        : proc.select_olccd_property,
```

Reply:
This change should fix the CI error

Reply:
Thanks very much! I did not manage to investigate ;smile: